### PR TITLE
Do some refactorings

### PR DESCRIPTION
**Pull request by atd-schubert**
These changes I had refactored:

* Use the most matching image (openjdk)
* I try with alpine (it makes your image much smaller)
* Do not use multiple `RUN` statements if not needed
* Clean-up before next `RUN` statement
* Register environment variables (`ENV`) at the beginning

Also I added a `docker-compose` file to make the build and execute process easier.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,26 +1,17 @@
-FROM debian:sid
```
Comment (atd-schubert) [inline Dockerfile:1]: You can use `openjdk-jre` or like I does to make the image smaller the alpine one.

The advantage is that you don't have to configure the JRE!

File: Dockerfile [hunk 2]
```diff
@@ -1,26 +1,17 @@
...
+FROM openjdk:8-jre-alpine
 
 MAINTAINER Christian Kuntzsch <christian@kuntzsch.me>
 
-RUN \
-    apt-get update && \
```
Comment (atd-schubert) [inline Dockerfile:6]: If you run `apt-get update` you should remove the package information after that to keep the image as small as possible...

File: Dockerfile [hunk 3]
```diff
@@ -1,26 +1,17 @@
...
-    apt-get install -y openjdk-8-jre wget
-
-ENV JAVA_HOME /usr/lib/jvm/java-8-openjdk-amd64
```
Comment (atd-schubert) [inline Dockerfile:9]: If you use the openjdk image you don't have to set these environment variables...

File: Dockerfile [hunk 4]
```diff
@@ -1,26 +1,17 @@
...
-
-RUN \
-    mkdir -p /var/otp && \
```
Comment (atd-schubert) [inline Dockerfile:12]: Register the environment variable before and then use it as target for the download.

File: Dockerfile [hunk 5]
```diff
@@ -1,26 +1,17 @@
...
-    wget -O /var/otp/otp.jar https://repo1.maven.org/maven2/org/opentripplanner/otp/1.2.0/otp-1.2.0-shaded.jar && \
-    wget -O /var/otp/jython.jar http://search.maven.org/remotecontent?filepath=org/python/jython-standalone/2.7.0/jython-standalone-2.7.0.jar
-
 ENV OTP_BASE /var/otp
-ENV OTP_GRAPHS /var/otp/graphs
-
-RUN \
-    mkdir -p /var/otp/scripting && \
```
Comment (atd-schubert) [inline Dockerfile:20]: You don't have to run `mkdir -p /var/otp` when you also run this command...

File: Dockerfile [hunk 6]
```diff
@@ -1,26 +1,17 @@
...
-    mkdir -p /var/otp/graphs/brandenburg && \
-    wget -O /var/otp/graphs/brandenburg/brb-gtfs.zip https://transitfeeds.com/p/verkehrsverbund-berlin-brandenburg/213/latest/download && \
-    wget -P /var/otp/graphs/brandenburg http://download.geofabrik.de/europe/germany/brandenburg-latest.osm.pbf && \
+ENV OTP_GRAPHS $OTP_BASE/graphs
```
Comment (atd-schubert) [inline Dockerfile:6]: Use also in the environment variable declarations the environment variables you registered before... Then you can make changes very easy and fast...

**Comment by atd-schubert** [comment]
Nice idea for a simple docker image. I just wanted to give you some hints, from the docker best-practices...

**Comment by chringel21** [comment]
Thanks for all the input! I took some notes!

**Pull request merged**